### PR TITLE
tests: add jammy pro test support (SC-868)

### PR DESCRIPTION
**Pull request by lucasmoura**
## Proposed Commit Message
tests: add jammy pro test support

## Test Steps
Run new the jammy pro tests

## Desired commit type

 - [ ] Squash and merge: Using the "Proposed Commit Message"
 - [ ] Create a merge commit and use "Proposed Commit Message"
 - [x] Rebase and merge, no merge commit, rely only on existing commit messages

## Checklist

 - [ ] I have updated or added any unit tests accordingly
 - [x] I have updated or added any integration tests accordingly
 - [ ] I have updated or added any documentation accordingly

### CODE REVIEW

File: features/ubuntu_pro.feature [hunk 1]
```diff
@@ -336,16 +337,69 @@ Feature: Command behaviour when auto-attached in an ubuntu PRO image
...
-            """
+        """
+        .*status=0\/SUCCESS.*
+        """
 
         Examples: ubuntu release
            | release | fips-s   | cc-eal-s | cis-s    | infra-pkg | apps-pkg | cis_or_usg |
            | xenial  | disabled | disabled | disabled | libkrad0  | jq       | cis        |
            | bionic  | disabled | disabled | disabled | libkrad0  | bundler  | cis        |
            | focal   | disabled | n/a      | disabled | hello     | ant      | usg        |
```
Comment (CalvoM) [inline features/ubuntu_pro.feature:348]: Is there no inclusion of jammy in this list?

File: features/ubuntu_pro.feature [hunk 2]
```diff
@@ -604,12 +658,12 @@ Feature: Command behaviour when auto-attached in an ubuntu PRO image
...
+        """
+        .*status=0\/SUCCESS.*
+        """
 
         Examples: ubuntu release
            | release | fips-s   | cc-eal-s | cis-s    | infra-pkg | apps-pkg | livepatch | cis_or_usg |
            | xenial  | n/a      | disabled | disabled | libkrad0  | jq       | n/a       | cis        |
-           | bionic  | disabled | disabled | disabled | libkrad0  | bundler  | n/a       | cis        |
+           | bionic  | disabled | disabled | disabled | libkrad0  | bundler  | enabled   | cis        |
            | focal   | disabled | n/a      | disabled | hello     | ant      | enabled   | usg        |
```
Comment (CalvoM) [inline features/ubuntu_pro.feature:669]: same question..

File: features/ubuntu_pro.feature (hunk 1 above)
Comment (orndorffgrant) [reply features/ubuntu_pro.feature:347]: @lucasmoura  would it be better or worse to break the enabling `usg`/`cis` on pro into it's own test that doesn't include jammy? Then the main pro test could be used for all series without duplicating

Comment (lucasmoura) [reply features/ubuntu_pro.feature:347]: @orndorffgrant just noticed that we also enable the `cis|usg` on the proxy tests. What if we keep it that only on those tests then ?

Comment (orndorffgrant) [reply features/ubuntu_pro.feature:347]: @lucasmoura good point, that works for me!

**Comment by lucasmoura** [comment]
@orndorffgrant updated

**Pull request merged**